src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -80,20 +62,15 @@ public void equals() {
         // different email -> returns false
         editedAlice = new PersonBuilder(ALICE).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
-
-        // different address -> returns false
-        editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).build();
-        assertFalse(ALICE.equals(editedAlice));
-
-        // different tags -> returns false
-        editedAlice = new PersonBuilder(ALICE).withTags(VALID_TAG_HUSBAND).build();
-        assertFalse(ALICE.equals(editedAlice));
     }
 
     @Test
     public void toStringMethod() {
-        String expected = Person.class.getCanonicalName() + "{name=" + ALICE.getName() + ", phone=" + ALICE.getPhone()
-                + ", email=" + ALICE.getEmail() + ", address=" + ALICE.getAddress() + ", tags=" + ALICE.getTags() + "}";
+        String expected = Person.class.getCanonicalName()
+                + "{name=" + ALICE.getName() + ", classGroup=" + ALICE.getClassGroup()
+                + ", email=" + ALICE.getEmail() + ", phone=" + ALICE.getPhone() + ", telegram="
+                + (ALICE.getTelegram().isPresent() ? ALICE.getTelegram().get().telegramId : "") + ", github="
+                + (ALICE.getGithub().isPresent() ? ALICE.getGithub().get().githubId : "") + "}";
```

Comment:
Referring to my suggestions made in `Telegram.java` and `Github.java`, this would be shortened to
`ALICE.getGithub().orElse(Github.EMPTY)`. The `toString()` invocation can be left out since string concatenation would call it implicitly